src/frontend/src/layouts/dm-layout/components/dm-side-bar/index.tsx

```diff
@@ -1,13 +1,30 @@
-type DmSideBarProps = {
-  serverName?: string
-}
+import MenuButton from '@/components/menu-button'
+import { useState } from 'react'
+
+function DmSideBar() {
+  const [selectedMenu, setSelectedMenu] = useState<string>('친구')
```

Comment:
initial 값으로 문자열을 넣어두었으면, 자동으로 추론하여 `string` 타입을 갖게됩니다. 삭제해도 될 듯 하네요
```suggestion
  const [selectedMenu, setSelectedMenu] = useState('친구')
```